agent/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/aws/amazon-ecs-agent/agent
 go 1.19
 
 require (
+	github.com/aws/amazon-ecs-agent/ecs-agent v0.0.0-00010101000000-000000000000
```

Comment:
nit: was this go.mod auto-generated via `go mod` (I assume it was)?
can this just be `v0.0.0` since we're replacing it anyway?

Reply:
Yeah this was auto-generated. I will simplify this.